dali/aux/optical_flow/optical_flow_adapter.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_AUX_OPTICAL_FLOW_OPTICAL_FLOW_ADAPTER_H_
+#define DALI_AUX_OPTICAL_FLOW_OPTICAL_FLOW_ADAPTER_H_
+
+#include <dali/pipeline/data/backend.h>
+#include "dali/kernels/tensor_view.h"
+
+namespace dali {
+namespace optical_flow {
+
+enum VectorGridSize {
+  UNDEF,
+  SIZE_4,  /// 4x4 grid
```

Comment:
For now I would stick to UNDEF or scrap this.

Reply:
Done